custom_dc/deploy_cloud.md

```diff
@@ -52,68 +52,65 @@ This procedure creates a "dev" Docker package that you upload to the Google Clou
 
    When prompted to confirm creating the credentials file, click `Y` to accept.
 
-1. Build a target package from the source image in the previous step. The `_ARTIFACT_REPO`_ must be an Artifact Registry repository you have created previously. The `_IMAGE_NAME`_ may be the same as the source (`datacommons-website-compose`) or any other string. The _`TARGET_IMAGE_TAG`_ can be the same as the source or any other string.
+1. Create a package from the source image created in step 1:
 
- <pre> 
-docker tag datacommons-website-compose:<var>DOCKER_TAG</var> \  
-<var>LOCATION</var>-docker.pkg.dev/<var>PROJECT_ID</var>/<var>ARTIFACT_REPO</var>/<var>IMAGE_NAME</var>:<var>TARGET_IMAGE_TAG</var>  
-</pre>
+    <pre> 
+   docker tag datacommons-website-compose:<var>DOCKER_TAG</var> \  
```

Comment:
I'm wondering if we should start using `datacommons-services` instead of `datacommons-website-compose` hereon and in prep for the new services container that will be using that name? Or wait to make this change when we update the doc for it?

Reply:
I'm more than happy to make the changes now, if you like.

Reply:
Thought a bit about it - let's change this when we update the doc to use the new image.